topology/sof-apl-nocodec.m4

```diff
@@ -211,23 +221,28 @@ DAI_ADD(sof/pipe-dai-capture.m4,
 	48, 1000, 0, 0)
 
 # PCM Low Latency, id 0
+dnl PCM_DUPLEX_ADD(name, pipeline, pcm_id, dai_id, playback, capture)
```

Comment:
dai_id: did you mean link_id?

Reply:
@plbossart this is actually not the link_id. 
But I'm making some more clean up for this. 

I think we should avoid the dai_id in the pcm definition. When creating cpu_dai's for the FE dai's, the id should come from the component->num_dai just like the cpu_dai id for the BE DAI's.

My next pull request will be about this change.